dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -249,7 +255,7 @@ private void handlePotentialFormatEntry(Set<FileFormat> fileFormats, Object v) {
     public Optional<String> getContent(String mainDescriptorPath, String mainDescriptor, Set<SourceFile> secondarySourceFiles, LanguageHandlerInterface.Type type,
         ToolDAO dao) {
         Yaml yaml = new Yaml();
-        if (isValidCwl(mainDescriptor, yaml)) {
```

Comment:
Why aren't you using this any more? It still gets used in 2 other spots, and it looks like you duplicate the code on line 273. I guess because you're throwing exceptions here and don't want to otherwise. Still, it would be nice if there wasn't the duplication... What happens we we support v2 some day, if there ever is one? If we don't update in both places....

Reply:
As you stated, I duplicated the code segment to allow for detailed exception messages. To address the versioning change, added static variable `CWL_VERSION_PREFIX` [here](https://github.com/dockstore/dockstore/blob/c5277c33e2a3bf271f3be4e33b742fb646b1a076/dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java#L71), making the version easier to update.